src/main/java/org/folio/circulation/domain/validation/BlockRenewalValidator.java

```diff
@@ -0,0 +1,59 @@
+package org.folio.circulation.domain.validation;
+
+import static org.folio.circulation.domain.RequestStatus.OPEN_NOT_YET_FILLED;
+import static org.folio.circulation.domain.RequestType.RECALL;
+import static org.folio.circulation.support.HttpResult.of;
+import static org.folio.circulation.support.HttpResult.succeeded;
+import static org.folio.circulation.support.ValidationErrorFailure.failure;
+
+import java.util.Comparator;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.circulation.domain.Item;
+import org.folio.circulation.domain.Request;
+import org.folio.circulation.domain.RequestQueue;
+import org.folio.circulation.domain.RequestQueueRepository;
+import org.folio.circulation.support.HttpResult;
+import org.folio.circulation.support.ValidationErrorFailure;
+
+public class BlockRenewalValidator {
+  private final RequestQueueRepository requestQueueRepository;
+
+  public BlockRenewalValidator(RequestQueueRepository requestQueueRepository) {
+
```

Comment:
Why is there a blank line here? Are you following a convention or is it a stylistic convention?

Reply:
I can't say that it improves readability or it is a standard. Shell I remove it?

Reply:
I would personally remove any of the blank lines that have been added if the white space doesn’t help readability by separating aspects.

Reply:
Done